tree/dataframe/src/RDFActionHelpers.cxx

```diff
@@ -204,10 +206,11 @@ void StdDevHelper::Finalize()
       if (fCounts[i] == 0) {
          continue;
       }
-      variance += fCounts[i] * (fDistancesfromMean[i] / fCounts[i] + std::pow((fMeans[i] - overallMean), 2));
+      auto setVariance = fDistancesfromMean[i] / (fCounts[i] - 1);
```

Comment:
@stwunsch 
Here the Welford's algorithm produces an unbiased variance for a subset, as in
`(mean, variance, sampleVariance) = (mean, M2/count, M2/(count - 1)) ` from [wiki](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Online_algorithm).